docs/source/conf.py

```diff
@@ -0,0 +1,217 @@
+# You can specify multiple suffix as a list of string:
+#
+source_suffix = ['.rst', '.md']
+autosectionlabel_prefix_document = True
+
+# The encoding of source files.
+#
+# source_encoding = 'utf-8-sig'
+
+# The master toctree document.
+master_doc = 'index'
+
+# General information about the project.
+project = 'Scylla Documentation Theme and Extensions'
+copyright = str(date.today().year) + ', ScyllaDB. All rights reserved.'
+author = u'Scylla Project Contributors'
+
+# The version info for the project you're documenting, acts as replacement for
+# |version| and |release|, also used in various other places throughout the
+# built documents.
+#
+# The short X.Y version.
+version = u'1.3'
+# The full version, including alpha/beta/rc tags.
+release = u'1.3.0'
+
+# The language for content autogenerated by Sphinx. Refer to documentation
+# for a list of supported languages.
+#
+# This is also used if you do content translation via gettext catalogs.
+# Usually you set "language" from the command line for these cases.
+language = None
+
+# List of patterns, relative to source directory, that match files and
+# directories to ignore when looking for source files.
+# This patterns also effect to html_static_path and html_extra_path
+exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store', 'lib', 'lib64','**/_common/*', 'README.md', '.git', '.github', '_utils', '_templates', 'rst_include']
+
+# The name of the Pygments (syntax highlighting) style to use.
+pygments_style = 'sphinx'
+
+# If true, `todo` and `todoList` produce output, else they produce nothing.
+todo_include_todos = True
+
+# Setup Sphinx
+def setup(sphinx):
+    sphinx.add_config_value('recommonmark_config', {
+        'enable_eval_rst': True,
+        'enable_auto_toc_tree': False,
+    }, True)
+    sphinx.add_transform(AutoStructify)
+
+# Adds version variables for monitoring and manager versions when used in inline text
+rst_prolog = """
+.. |mon_version| replace:: 3.1
+.. |man_version| replace:: 2.0
+.. |mon_root| replace::  :doc:`Scylla Monitoring Stack </operating-scylla/monitoring/index>`
+""" 
+
+# -- Options for HTML output ----------------------------------------------
+
+# The theme to use for HTML and HTML Help pages.  See the documentation for
+# a list of builtin themes.
+#
+html_theme = 'sphinx_scylladb_theme'
+html_theme_path = ["../.."]
+html_style = ''
+
+# Theme options are theme-specific and customize the look and feel of a theme
+# further.  For a list of options available for each theme, see the
+# documentation.
+#
+
+html_theme_options = {
+    'header_links': [
+    ('Scylla Rust Driver', 'https://scylladb.github.io/rust-driver/'),
+    ('Scylla Cloud', 'https://docs.scylladb.com/scylla-cloud/'),
+    ('Scylla University', 'https://university.scylladb.com/'),
+    ('ScyllaDB Home', 'https://www.scylladb.com/')],
+    'site_description': 'Scylla Driver for Rust.',
+    'github_issues_repository': 'scylladb/scylla-rust-driver',
+    'show_sidebar_index': 'false',
```

Comment:
Set ``show_sidebar_index`` to ``'true'``.  This change will render the sidebar in the index page.

For example:

```
 'show_sidebar_index': 'true',
```